build/scripts/build-volplugin-containers.sh

```diff
@@ -1,4 +1,12 @@
 #!/bin/bash
+
+# In order for the new binaries to take effect we need to clean the older environment
+# Otherwise the binaries stick to older container images
+# repeated docker builds also lead to dangling images, which need to be cleaned too.
+docker ps -a | grep -e volplugin -e apiserver -e volsupervisor | awk '{print $1}' | xargs docker rm -fv
+docker rmi contiv/volplugin
+docker rmi $(docker images -f "dangling=true" -q)
```

Comment:
yeah, I read up about this, so there are two type of dangling images, ones that show up when doing `docker ps -a` are the good ones and should not be removed. They are like the empty parents to existing good images. The ones that show up with the `-a` option are the bad ones. I have verified this locally on my setup multiple times.